pyproject.toml

```diff
@@ -14,6 +14,9 @@ authors = [
 requires-python = ">=3.8"
 keywords = ["disnake", "discord", "discord api"]
 license = { text = "MIT" }
+dependencies = [
+    'aiohttp>=3.7.0,<4.0',
```

Comment:
What happened to the `python_version < "3.11"` env markers?

Reply:
I have to double check pep 621 and pep 508 to see if we can have two declarations of the same dependency with different markers.

Reply:
Should be valid

Reply:
This is a bug in resolvelib which is the same lib that pdm also uses.

https://github.com/pdm-project/pdm/issues/46

Reply:
I also don't think that this is worth blocking the whole pr over. Its a minor change that we made as a convenience to the users